### PR TITLE
what is dead may never die

### PR DESCRIPTION
**Pull request by gurneyalex**
but rises again, harder and stronger

don't try to delete what we have not found.

### CODE REVIEW

**Comment by yvaucher** [comment]
:+1:

File: features/steps/dsl.py [hunk 1]
```diff
@@ -140,7 +140,8 @@ def impl(ctx, company_oid):
 
 @step('I delete it')
 def impl(ctx):
-    ctx.found_item.unlink()
+    if ctx.found_items is not None:
```
Comment (lepistone) [inline features/steps/dsl.py]: remove the `s`
or anyway, use the same thing.

**Comment by lepistone** [comment]
also, I am not sure in all steps we put exactly `None` if nothing found. Who knows, we might use `False` or empty recordlists or something. Let's check for true-ish-ness.

**Comment by lepistone** [comment]
thanks :+1:

**Comment by guewen** [comment]
:pray:

**Comment by yvaucher** [comment]
Good catch @lepistone to use found_items, we could add a sentence `I delete them`

**Pull request merged**